bench/workgen/workgen.cpp

```diff
@@ -382,7 +388,7 @@ WorkloadRunner::create_table(WT_SESSION *session, const std::string &config, con
         icontext->_dyn_tint[uri] = tint;
         icontext->_dyn_table_names[tint] = uri;
         icontext->_dyn_table_runtime[tint] = TableRuntime(is_base, mirror_uri);
-        ++icontext->_dyn_tint_last;
+        (void)workgen_atomic_add32(&icontext->_dyn_tint_last, 1);
```

Comment:
do we not want to retry or error out if this fails?

Reply:
Can this fail?

Reply:
If there are multiple threads accessing this it can, but here I can see it is protected by a lock so it should be okay? I am not sure if we still need atomic if theres already a lock.

Reply:
There is a lock so we should be safe in this regard! And you are right, we don't need the atomic call since we have a lock.